src/code_d0a70_len_4fe0.c

```diff
@@ -14,7 +14,14 @@ INCLUDE_ASM(s32, "code_d0a70_len_4fe0", func_8013A6E8);
 
 INCLUDE_ASM(s32, "code_d0a70_len_4fe0", func_8013A704);
 
-INCLUDE_ASM(s32, "code_d0a70_len_4fe0", func_8013A854);
+void func_8013A854(u32 i) {
+    UnkStruct8013A854* unkStruct80156954ListPtr;
```

Comment:
Empty line after declaration please

Reply:
After the `void func_8013A854(u32 i) {` declaration right?

```

void func_8013A854(u32 i) {
    // <------- here
    UnkStruct8013A854* unkStruct80156954ListPtr;
    if (i < 90) {
        unkStruct80156954ListPtr = *D_80156954;
        unkStruct80156954ListPtr[i].unk_14 = 0;
        unkStruct80156954ListPtr[i].unk_10 = -1;
    }
}

```